src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx

```diff
@@ -1,225 +1,177 @@
-        label: 'Rewind',
-        spellId: TALENTS_EVOKER.REWIND_TALENT.id,
-        value: this.healingForSpell(TALENTS_EVOKER.REWIND_TALENT.id),
-        valueTooltip: formatNumber(this.healingForSpell(TALENTS_EVOKER.REWIND_TALENT.id)),
-      },
-      {
-        color: SPELL_COLORS.CYCLE_OF_LIFE,
-        label: 'Cycle of Life',
-        spellId: SPELLS.CYCLE_OF_LIFE_HEAL.id,
-        value: this.healingForSpell(SPELLS.CYCLE_OF_LIFE_HEAL.id),
-        valueTooltip: formatNumber(this.healingForSpell(SPELLS.CYCLE_OF_LIFE_HEAL.id)),
-      },
-      {
-        color: SPELL_COLORS.EMERALD_COMMUNION,
-        label: 'Emerald Communion',
-        spellId: TALENTS_EVOKER.EMERALD_COMMUNION_TALENT.id,
-        value:
-          this.healingForSpell(SPELLS.EMERALD_COMMUNION_ALLY.id) +
-          this.healingForSpell(TALENTS_EVOKER.EMERALD_COMMUNION_TALENT.id),
-        valueTooltip: formatNumber(
-          this.healingForSpell(SPELLS.EMERALD_COMMUNION_ALLY.id) +
-            this.healingForSpell(TALENTS_EVOKER.EMERALD_COMMUNION_TALENT.id),
-        ),
-      },
-      {
-        color: '#828282',
-        label: 'Other',
-        spellId: 0,
-        value: this.healingForSpell(0),
-        valueTooltip:
-          formatNumber(this.healingForSpell(0)) +
-          ' (This includes misc items, trinkets, and other sources of non-spell healing)',
-      },
-    ]
-      .filter((item) => {
-        return item.value > 0 && item.value >= this.healingForSpell(0);
-      })
-      .sort((a, b) => {
-        return Math.sign(b.value - a.value);
-      });
-    return <DonutChart items={items} />;
+  getTotalSelfHealing(window: HealEvent[]) {
+    let totalSelfHealing = 0;
+    for (const Heal of window) {
+      totalSelfHealing += Heal.amount + (Heal.absorbed || 0) + (Heal.overheal || 0);
+    }
+    return totalSelfHealing;
   }
 
   statistic() {
+    console.log(this.lifebindWindows);
+
     return (
       <Statistic
-        position={STATISTIC_ORDER.CORE(5)}
+        position={STATISTIC_ORDER.OPTIONAL(13)}
         size="flexible"
         category={STATISTIC_CATEGORY.TALENTS}
-      >
-        <div className="pad">
-          <label>
-            <SpellLink spell={TALENTS_EVOKER.LIFEBIND_TALENT} />{' '}
-            <small>healing breakdown by spell</small>
+        wide
+        tooltip={
+          <>
+            A <SpellLink spell={TALENTS_EVOKER.LIFEBIND_TALENT} /> window is when you consume
+            atleast five <SpellLink spell={TALENTS_EVOKER.ECHO_TALENT} />
+            es with <SpellLink spell={TALENTS_EVOKER.VERDANT_EMBRACE_TALENT} /> in order to apply{' '}
+            <SpellLink spell={TALENTS_EVOKER.LIFEBIND_TALENT} /> to a big part of your group and
+            then do self healing on yourself to transfer it via the{' '}
+            <SpellLink spell={TALENTS_EVOKER.LIFEBIND_TALENT} />.
+            <br />
+            <br />
+            At the moment the best way for Chronowarden to do healing via{' '}
```

Comment:
You're providing Chronowarden specific advice that will be enabled for Flameshaper. You should probably gate by hero talent for this part of it or change the language to be more broad.